app/modules/individuals/resources.py

```diff
@@ -43,16 +43,20 @@ def __init__(self):
     def rollback_and_abort(self, message='Unknown Error', code=400):
         if self.individual_guid is not None:
             failed_individual = Individual.query.get(self.individual_guid)
+            # we try deleting from edm *regardless of failed_individual*, as sometimes
+            #   the houston individual did not get made, but the edm did
+            log.error(f'Attempting to delete Individual {self.individual_guid} from EDM')
```

Comment:
Not sure about the answer but should this be an audit log instead?

Reply:
i am going to go with no, just cuz this is cleanup of an _aborted_ attempt and these objects were only momentarily there.